README.md

```diff
@@ -7,4 +7,16 @@ A general library that is supposed to be used in most PHP implementations of H5P
 
 All code is licensed under MIT License
 
-Open Sans font is licensed under Apache license, Version 2.0
\ No newline at end of file
+Open Sans font is licensed under Apache license, Version 2.0
+
+## Compiling scss to css
+
+You first need to install ruby and compass:
+- `sudo apt update`
+- `sudo apt install ruby-full`
+- `sudo apt install build-essential`
+- `sudo gem install compass`
+
+Then cd to `h5pcom-editor-php-library/syles` and compile the scss files:
```

Comment:
```suggestion
Then cd to `h5p-editor-php-library/styles` and compile the scss files:
```

Reply:
@boyum I assume there's a "shadow repository" just for H5P.com, so maybe both depending on where one is working on? `syles` should be `styles` either way :-)

Reply:
Yeah, but as this is a public repository and there are no mentions of cloning the repo as `h5pcom-editor-php-library`, I'm sure this would cause more confusion than it would help.

Reply:
@boyum Agreed. As an external observer, one can get the impression that internally H5P is more and more a synonym for H5P.com and the open source nature is losing relevance.

Reply:
@otacke That is sad :(

Reply:
@boyum Thanks for noticing! I've applied your correction, thank you.